src/event.rs

```diff
@@ -31,7 +33,7 @@ pub struct LeafInfo<TYPES: NodeType> {
     /// Optional application-specific state delta.
     pub delta: Option<Arc<<<TYPES as NodeType>::ValidatedState as ValidatedState<TYPES>>::Delta>>,
     /// Optional VID disperse data.
-    pub vid: Option<VidDisperse<TYPES>>,
+    pub vid: Option<VidDisperseShare<TYPES>>,
```

Comment:
This is a tricky one, I think this changes seems more correct, but maybe the sequencer would want more than one share, if we had it.  We should check what they do with the `VidDisperse` now.  I think they just get their share and this change will simplify their code

Reply:
I'm not sure, we can just ask them though